ClientSideEncryptionToCPKN/localKeyClientSideToCustomerManagedServerSide/samplesrc/Migration.cs

```diff
@@ -0,0 +1,130 @@
+﻿using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+using Azure.Storage;
+using Azure.Core;
+using Azure.Identity;
+using System;
+using System.IO;
+using System.Configuration;
+
+namespace localKeyClientSideToCustomerManagedServerSide
+{
+    class Migration
+    {        
+        private static void CSEtoSSE(
```

Comment:
Add summary/comment to start of function and rename method to `EncryptionScopeWithCustomerManagedKey` something like that